src/main/java/wanion/avaritiaddons/block/chest/infinity/TileEntityInfinityChest.java

```diff
@@ -133,13 +138,13 @@ public ItemStack getStackInSlot(final int slot)
 	@Override
 	public boolean isItemValidForSlot(final int slot, final ItemStack itemStack)
 	{
-		return slot != 243;
+		return slot != 243 || findSlotFor(itemStack) != -1;
 	}
```

Comment:
a bit hacky way to check if the slot is correct for item, but I guess this is the way AE does it, right? 

Better solution would be for chest to manage slot finding/checking it self with method like `boolean accept(ItemStack stack)` or `boolean canAccept(ItemStack stack)` or `int findSlotFor(itemStack)` or `Optional<Integer> findSLotFor(itemStack)`. (if you don't like -1 hack) Is there a way to change this or is it buried too deep to be worth changing

Reply:
i haven't fully understand the inf chest tbh. the code logic is very confusing in this place, so i only can copy the old code which runs well to prevent further bugs. let's leave it alone